itests/kit2/init.go

```diff
@@ -0,0 +1,29 @@
+package kit2
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/filecoin-project/go-state-types/abi"
+	"github.com/filecoin-project/lotus/build"
+	"github.com/filecoin-project/lotus/chain/actors/policy"
+	logging "github.com/ipfs/go-log/v2"
+)
+
+func init() {
+	_ = logging.SetLogLevel("*", "INFO")
+
+	policy.SetConsensusMinerMinPower(abi.NewStoragePower(2048))
+	policy.SetSupportedProofTypes(abi.RegisteredSealProof_StackedDrg2KiBV1)
```

Comment:
Suggest adding 8MB proof types as well so we can run integration tests with 2k or 8MB sector sizes

Reply:
Makes sense. Feel free to push the change directly!